### DISCUSSION TITLE
Why the Ctrl + Q hotkey doesn't quit the application?

### DISCUSSION THREAD

**Discussion by ghost** [andyholmes/gnome-shell-extension-gsconnect#1670] [Q&A]
The said shortcut doesn't appear to have an effect. Alt + F4 can be used, but I'm not certain if the combination sends a "polite" shutdown request on Linux.

Of the top of my head, I can't recall any graphical program which doesn't close when pressing Ctrl + Q, outside of Gnome Tweaks. Some Gnome windows (e.g. sub-menus of settings) and in-house  applications may not react to Ctrl +Q or Ctrl + W.

I'm referring to the Gsconnect's 'messaging' and 'mobile settings' components, I haven't used the other features extensively. 

    gnome-shell-extension-gsconnect.x86_64               55-1.fc38                           @fedora

**Comment by andyholmes** [comment]
The messaging window is a part of the service process, while the preferences window is a discrete process. I'm not sure either ever had support for those keybindings added.
